tagger.py

```diff
@@ -0,0 +1,57 @@
+import os
+import pydicom
+import argparse
+
+
+class TagError(Exception):
+    pass
+
+
+def add_private_tag(dicomfile, group, identifier, tag_value):
+    for elem_tag in range(0x0010, 0x00FF):
+        dataelem = dicomfile.get((group, elem_tag))
+        if dataelem:
+            is_private_creator = dataelem.tag.group == group and 0x0010 <= dataelem.tag.element <= 0x00ff
```

Comment:
Under what circumstances is this `False`? Does pydicom return a different element than you asked for?

Reply:
It's a check to see if that tag is free, if there is no element there it returns None

Reply:
Question is about Line 14: `is_private_creator`

Reply:
Ah, yeah that should be removed